openpathsampling/analysis/shooting_point_analysis.py

```diff
@@ -107,7 +118,16 @@ def analyze(self, steps):
             MC steps to analyze
         """
         for step in self.progress(steps):
-            self.analyze_single_step(step)
+            try:
+                self.analyze_single_step(step)
+            except NoFramesInStateError as err:
+                if self.error_if_no_state:
+                    addition = ("\nTo disable this error set "
+                                "'error_if_no_state=False'")
+                    raise type(err)(str(err) + addition)
+
+                else:
+                    pass
```

Comment:
So, the code now warns once per `mccycle` as normally warnings are filtered per message and the MC cycle is the number that is updated. Would that be sufficient for now?

Reply:
Yep, once per `mccycle` is what I meant. (Since this is based on trial trajectories from shooting moves, that's essentially the same as once per trajectory, but you're right that the loop is over steps/`mcccycle`.)